pages/common/histexpand.md

```diff
@@ -1,6 +1,4 @@
-# bash-histexpand
-
-> Reuse and expand your bash history.
+> Reuse and expand your history in bash or zsh.
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
> Reuse and expand the shell history in bash or zsh.
```